src/main/java/pl/gispartner/ResourceManagerApp/api/UserController.java

```diff
@@ -0,0 +1,28 @@
+package pl.gispartner.ResourceManagerApp.api;
+
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+import pl.gispartner.ResourceManagerApp.model.UserEntity;
+import pl.gispartner.ResourceManagerApp.service.UserService;
+
+@RestController
+@RequestMapping("/api/user")
+@RequiredArgsConstructor
+public class UserController {
+
+    private final UserService userService;
+
+    @PostMapping("/save")
+    public void saveUser(@RequestBody UserEntity userEntity){
+        userService.saveUser(userEntity);
+    }
+    @DeleteMapping("/delete")
+    public void deleteUser (@RequestBody UserEntity userEntity){
+        userService.deleteUser(userEntity);
+    }
+    @PutMapping("/update")
+    public void updateUserNickName(@RequestBody UserEntity userEntity){
```

Comment:
jw. /update zbedne, jesli aktualizujesz konkretna rzecz (nick) to wystarczy ze tylko to pole podasz jako body, nie podawaj calego obiektu jesli aktualizujesz jedno pole